client/src/main/java/org/eclipse/hono/client/DownstreamSender.java

```diff
@@ -162,6 +166,7 @@
      * @throws IllegalArgumentException if the content type specifies an unsupported character set.
      */
     Future<ProtonDelivery> send(
+            Long maxTtl,
```

Comment:
I'd rather not introduce a new parameter here but instead simply use the `MessageSender.send(Message)` method for sending the event ...

Reply:
I was thinking about this too, but then the code that sets some of properties of the message (body, content-type, to, ...) will be duplicated in the two places. will fix